sdk/nodejs/api/api.ts

```diff
@@ -0,0 +1,447 @@
+import { GraphQLClient, gql } from "../index.js";
+import { 
+  ContainerExecArgs, 
+  ContainerWithFsArgs, 
+  ContainerWithMountedDirectoryArgs, 
+  ContainerWithWorkdirArgs, 
+  DirectoryEntriesArgs, 
+  DirectoryFileArgs, 
+  GitRepositoryBranchArgs, 
+  HostWorkdirArgs, 
+  QueryContainerArgs, 
+  QueryGitArgs, 
+  Scalars } from "./types.js";
+import { queryBuilder, queryFlatten } from "./utils.js"
+
+export type QueryTree = {
+  operation: string
+  args?: Record<string, any>
+}
+
+export class BaseClient {
```

Comment:
Do we need to export this?

Reply:
No you're right, this export is useless. Thanks